localstack/services/sqs/sqs_starter.py

```diff
@@ -112,7 +112,7 @@ def render(self, *args, **kwargs):
 
     # escape message responses to allow for special characters like "<"
     sqs_responses.RECEIVE_MESSAGE_RESPONSE = sqs_responses.RECEIVE_MESSAGE_RESPONSE.replace(
-        "<StringValue>{{ value.string_value }}</StringValue>",
+        "<StringValue><![CDATA[{{ value.string_value }}]]></StringValue>",
```

Comment:
Thanks for this fix @eltone ! 👍 

Wondering if we should make this a bit more resilient to upstream changes, e.g., via a regular expression. Not critical, though - we can pick that up in a follow-up PR.. Thanks